mark WakeLock.request() 's parameters as optional (#29797)
mdn/content: files/en-us/web/api/wakelock/request/index.md

### Parameters

- `type` {{optional_inline}}

  - : A screen wake lock type from among the following enumerated types:

    - `'screen'`
      - : Prevents devices from
        dimming or locking the screen.

> 

Reviewer: ```suggestion

If no `type` parameter is explicitly specified, the `request()` method defaults to using the `screen` type.

```

### Parameters

- `type` {{optional_inline}}

  - : A string specifying the screen wake lock type, from among the following enumerated types:

    - `screen`

>       - : Prevents devices from dimming or locking the screen.

Reviewer: ```suggestion
      - : Prevents the screen from turning off. Only visible documents can acquire the screen wake lock.
```

### Parameters

- `type` {{optional_inline}}

>   - : A screen wake lock type from among the following enumerated types:

Reviewer: ```suggestion
  - : A string specifying the screen wake lock type, from among the following enumerated types:
```

>     - `'screen'`

Reviewer: ```suggestion
    - `screen`
```

>       - : Prevents devices from
>         dimming or locking the screen.
> 

Reviewer: ```suggestion
      - : Prevents devices from dimming or locking the screen.

```

### Parameters

- `type` {{optional_inline}}

>   - : Default to be `'screen'`. Available values are as follows:

Reviewer: ```suggestion
  - : A screen wake lock type from among the following enumerated types:
```

    - `'screen'`

>       - : Requests a screen wake lock. Prevents devices from

Reviewer: ```suggestion
      - : Prevents devices from
```
